stream.go

```diff
@@ -70,7 +70,8 @@ type StreamDesc struct {
 
 	// ServerStreams and ClientStreams are used for registering handlers on a
 	// server as well as defining RPC behavior when passed to NewClientStream
-	// and ClientConn.NewStream.  At least one must be true.
+	// and ClientConn.NewStream.  At least one must be true. In case of
+	// ClientStreams=false for StreamDesc, nil is returned unconditionally.
```

Comment:
why did you change documentation here? The SendMsg Docstring needs to change only and there you need to add additional line